src/extension.ts

```diff
@@ -164,3 +177,31 @@ function convertLogLevel(commonLogLevel: string) {
             return "FINE";
     }
 }
+
+function resolvePotentialProjectFiles() {
+    let editors = vscode.window.visibleTextEditors.filter((e) => e.document.languageId === "java");
+    const activeEditor = vscode.window.activeTextEditor;
+    if (activeEditor.document.languageId === "java") {
```

Comment:
Maybe user doesn't open java file and just click F5, is there possible to throw null pointer exception?

Reply:
nice catch!